### DISCUSSION TITLE
Version 9.5

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#418] [Announcements]
* Added:
    * Visual improvements in several parts of the extension. Thanks to: CaganAgabey
    * Button to copy selected translated text
* Fixed:
    * Fixed: Donation button doesn't work
    * Bing translate does not work
    * Translate PDF with google translation only translates to Portuguese
    * Sometimes the translate selected text popup appears off screen
    * When translating selected text on some websites, there is no way to manually copy the text
    * Input of type reset is not translated
* Changes:
    * Now you can translate into about:blank
* Updated interface translations
* General fixes and improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.5'>Version 9.5</a>.</em>

**Comment by Cheshire-Behemoth** [comment]
Good afternoon! Tell me how to remove the transparency of the translation pop-up window when you select the text with the mouse and click on the button that appears. Due to the transparency of the background, the translation text is almost unreadable

**Reply by FilipePS** [reply]
Are you using a Chromium based browser?

**Reply by Cheshire-Behemoth** [reply]
Sorry for my English, I am writing a translator through Google. I use Firefox.

**Comment by Cheshire-Behemoth** [comment]
This is how it looks like
https://i.postimg.cc/BQYyLDZq/1.jpg

**Reply by FilipePS** [reply]
It's already been removed!

**Comment by FilipePS** [comment]
This is how it should look in Firefox (in chrome it's different).
![image](https://user-images.githubusercontent.com/54990796/174659864-0c1d5250-c1c5-4180-9083-183c19146b07.png)

What is your Firefox version?

**Reply by Cheshire-Behemoth** [reply]
101.0.1 64 bit

**Reply by FilipePS** [reply]
Did you edit any settings in `about:config`?

**Reply by Cheshire-Behemoth** [reply]
Yes, I made changes to this file, as in user.js
I also use the visual style https://github.com/black7375/Firefox-UI-Fix/tree/photon-style

**Reply by Cheshire-Behemoth** [reply]
But your application is the first where this problem appeared

**Reply by FilipePS** [reply]
The `layout.css.backdrop-filter.enabled` is enabled?

**Comment by Cheshire-Behemoth** [comment]
Yes

**Reply by FilipePS** [reply]
This effect is turned off by default. And in chrome enabled by default.

In chrome the popup looks like this:
![image](https://user-images.githubusercontent.com/54990796/174663702-bcc29cd0-0590-45fa-b005-074181fe32d4.png)

**Reply by Cheshire-Behemoth** [reply]
disabled this option, it became like this
https://i.postimg.cc/mDDYzdbj/1.jpg
Looks very ugly

**Reply by Cheshire-Behemoth** [reply]
All these games with translucency never bring anything good in terms of readability.

**Comment by Cheshire-Behemoth** [comment]
the best solution would be the ability to turn off transparency completely or add an adjustment to it
